pyk/src/pyk/testing/_kompiler.py

```diff
@@ -300,6 +302,24 @@ def kcfg_explore(
         yield KCFGExplore(cterm_symbolic, kcfg_semantics=semantics)
 
 
+class ParallelTest(KCFGExploreTest):
```

Comment:
Inheriting from `KCFGExploreTest` seems unnecessary here, as the fixtures defined there are not used.